### PR TITLE
added Moderator tag to ban message in messageCreate.ts

### PR DESCRIPTION
**Pull request by WWW0030**
# Related Issues

# Summary of Changes

# Steps to Reproduce

### CODE REVIEW

**Review by Picowchew — CHANGES_REQUESTED** [review]
Good start! I have a few comments.

It would be nice if you could fill out the PR description. There are comments in the PR template to describe what is meant to be written in each section. This makes it much easier for the reviewer to go through your PR. For instance, I was actually confused at first as to the purpose of this PR; was it supposed to DM the banned person with the moderator for them to contact or was it supposed to log which moderator (or Codey) did the banning. I later received clarification that you are working on this ticket https://github.com/uwcsc/codeybot/issues/185, so in the "Related Issues" section, you can do something like `Resolves #185` to automatically close the issue when the PR is merged (https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).

I believe that what the code currently does is that in the log message (which is sent in the server) that is only accessible by mods and executives, the username and discriminator of a mod is added to the reason. What should be done instead is that the banned person should receive a DM from Codey with a ping to a mod. In this way, the banned person would be able to contact a mod after leaving the server due to the ban. The reason why a ping is used is that the banned person can easily click on the ping and then message the mod. If they do not have any mutual servers though, then from the ping, the username and discriminator can be obtained, and then from this information, the banned person is able to send the mod a friend request. After accepting the friend request, the banned person is able to message the mod. The coffee chat code DMs people with a ping, so feel free to look there for some guidance on how this could be done. In addition, as described in the issue, you can add the ID of the mod as another config variable as opposed to making the username and discriminator a constant within the file. This allows for easier changing of the mod.

Feel free to reply here if you have any questions!

File: src/listeners/messageCreate.ts [hunk 1]
```diff
@@ -38,7 +39,7 @@ const punishSpammersAndTrolls = async (message: Message): Promise<boolean> => {
     await message.delete();
     if (message.member) {
       const user = message.member.user;
-      const reason = 'Spammer/troll/got hacked';
+      const reason = 'Spammer/troll/got hacked'+'for appeal, please contact: '+APPEAL_MODERATOR_ID;
```
Comment (alexz429) [inline src/listeners/messageCreate.ts:42]: make sure this looks right when you test it, there doesn't seem to be a space between hacked and for

File: src/listeners/messageCreate.ts [hunk 2]
```diff
@@ -7,6 +7,7 @@ import { sendKickEmbed } from '../utils/embeds';
 
 const HONEYPOT_CHANNEL_ID: string = vars.HONEYPOT_CHANNEL_ID;
 const ANNOUNCEMENTS_CHANNEL_ID: string = vars.ANNOUNCEMENTS_CHANNEL_ID;
```
Comment (alexz429) [inline src/listeners/messageCreate.ts:9]: check these linting errors, typically if you installed a linter into your IDE these should be autofixed.

**Review by alexz429 — CHANGES_REQUESTED** [review]

**Comment by mcpenguin** [comment]
Any updates on this?

**Comment by mcpenguin** [comment]
Closing this PR due to inactivity.

**Pull request closed without merging**